mypyc/test-data/irbuild-dunders.test

```diff
@@ -154,13 +154,21 @@ class C:
     def __abs__(self) -> int:
         return 6
 
+    def __bool__(self) -> bool:
+        return False
+
+    def __complex__(self) -> complex:
+        return 7+1j
```

Comment:
Just returning `1j` here should help with the test failures, or you can update the test stub in `mypyc/test-data/fixtures/ir.py`.

Reply:
I have no idea how to fix the stubs so I just updated the irbuild test to just return `7j`.